notebooks/en/annotate_text_data_transformers_via_active_learning.ipynb

```diff
@@ -0,0 +1,2296 @@
+{
```

Comment:
<p>It might be worth briefly explaining here how you are doing the additional annotations here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>@davanstrien I have explained it via a flow chart above and also added specific pointers above flow chart. </p>